docs/conf.py

```diff
@@ -173,6 +173,8 @@
 # included in the gallery, but its plot is available for
 # the quickstart
 sphinx_gallery_conf = {
+    "doc_module" : ('fairlearn.fairlearn.datasets', 'fairlearn.fairlearn.metrics', 'fairlearn.fairlearn.postprocessing', 'fairlearn.fairlearn.preprocessing',  'fairlearn.fairlearn.reductions', 'fairlearn.fairlearn.adversarial', 'fairlearn.fairlearn.experimental'),
+    "backreferences_dir" : "backref_dir",    
```

Comment:
this is what we have in sklearn though:

```py
    "doc_module": "sklearn",
    "backreferences_dir": os.path.join("modules", "generated"),
```

Reply:
the doc_module is supposed to contain the names of the modules for which documentation should be rebuilt. I gave those values to follow the folder structure for the project. I tried with "fairlearn.module_name" and just "fairlearn" but with no results.
The backreferences_dir is supposed to be a place where the generated gallery would be stored.  So it has to be initially empty. The value entered in scikit-learn (modules/generated) doesn't already exist in their project - only the "modules" folder exists. So I believe "generated" gets created when the documentation is rebuilt, although I'm not 100% sure about that.

Reply:
I think the right way to specify the doc module is just to specify "fairlearn". I tried building the docs using this conf.py file, and it looks like in the backref_dir, the thumbnails are being generated (see the image below), but they aren't showing up on the API pages. I wonder if that's because of the way that the API reference is now structured (i.e., using autodoc).

![image](https://github.com/fairlearn/fairlearn/assets/15962391/7691c9ab-4136-43bd-b7ba-bcc60df40dd4)